packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.js

```diff
@@ -360,6 +364,28 @@ export default class InlineContent extends Base {
         text: text.text || text
       }
     };
+    if (textOverrideStyles?.textDecoration === 'line-through') {
+      const textWidth = measureTextWidth({
+        ...this.style.textStyle,
+        ...textOverrideStyles,
+        text: text.text || text
+      });
+      const strikethroughLine = {
+        rect: true,
+        w: textWidth,
+        color: textOverrideStyles?.textColor || this.style.textDefaultColor,
```

Comment:
Sorry, one last thing!

Can we actually reverse this order and rename the style property to `strikethroughColor` (and update the style file, too)? I think that way makes more sense so that users can define a color they want the strikethrough to be, and then we provide fallbacks.

So I think this should be `color: this.style.strikethroughColor || textOverrideStyles?.textColor || this.style.textStyle.textColor`. That'll fall back to the text override first, and then the default text style.

Reply:
Sounds good. I just updated the property name and changed the order.